src/html/ReqMgr/templates/doc.tmpl

```diff
@@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
     }
     ShowTag('edit-'+tag);
 }
+function UpdateTaskParams() {
+    var parameters = \$('#taskParams').val();
+    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
```

Comment:
Seangchan,
parameters passed to ajaxRquest later on are JSON.stringify'ed, see
https://github.com/vkuznet/WMCore/blob/reqmgr2-tasks/src/html/ReqMgr/javascript/ajax_utils.js#L46

So I think everything is alright on my end. Could you please be more specific
about failures you saw.
Thanks,
Valentin.

On  0, ticoann <notifications@github.com> wrote:
> > @@ -86,8 +103,17 @@ function Activate(tag, hidetags) {
> >      }
> >      ShowTag('edit-'+tag);
> >  }
> > +function UpdateTaskParams() {
> > +    var parameters = \$('#taskParams').val();
> > +    ajaxRequest('$base/data/splitting/$rid', parameters, 'POST');
> 
> parameters doesn't seems to be json encoded
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6605/files#r53517786